doc/nrf/gs_installing.rst

```diff
@@ -55,6 +55,61 @@ The installation process is different depending on your operating system.
 
       Also see :ref:`zephyr:mac-setup-alts` for additional information.
 
+If you are interested in building `Connected Home over IP`_ applications, install also the `GN`_ meta-build system to generate files for `Ninja`_.
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      The Connected Home over IP project currenlty supports only Linux and macOS.
```

Comment:
I think we should start by saying that GN is not supported on Windows yet, because that is what the installation is about. And then go on to say that CHIP supports only Linux and macOS.

Reply:
Ok, I guess it's fixed.